validation/l2oo_test.go

```diff
@@ -19,33 +19,23 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
+type L2OOParams struct {
+	SubmissionInterval        *big.Int
+	L2BlockTime               *big.Int
+	FinalizationPeriodSeconds *big.Int
```

Comment:
Do we want to change the name of this variable to be challenge period? Maybe not as I know the on-chain function is `finalizationPeriodSeconds()`

Reply:
(So keep both names around, one on the data scraping side, one on the spec side).

Reply:
Just unresolved for visibility: It's a bit unfortunate there's two names, I'm indifferent to the name used as long as it's clear they are the same. Maybe just a comment in the toml file is sufficient:
```toml
challenge_period_seconds = [12, 12] # i.e. finalization period
```

Reply:
[9cb8ac3](https://github.com/ethereum-optimism/superchain-registry/pull/208/commits/9cb8ac3b779d189455dd9c46ad01439cf1edcc6a)